### ISSUE TITLE
walletrpc blockchain sync error

### ISSUE THREAD

**Issue by itswisdomagain** [raedahgroup/godcr-old#132]
Each time godcr is run in http mode, a blockchain sync is performed and the pages and features are disabled until sync is completed. Recently, sync does not seem to start or complete when connecting to dcrwallet rpc, and the web features remain unaccessible.

It's possible this issue only occurs on recent versions of dcrwallet binary. The issue was observed with the following dcrwallet and go versions
```
dcrwallet version 1.4.0-rc2+release (Go version go1.11.4)
```

Below is the screenshot of the issue, running `godcr --mode=http` with `usewalletrpc=1` set in config:

![screenshot 2019-01-06 at 5 11 41 pm](https://user-images.githubusercontent.com/18400051/50738821-6cf34080-11d8-11e9-903f-0c36be05d25f.png)

**Comment by barisere** [comment]
I can't reproduce this error. When running in HTTP mode, sync completes, as I can tell from the logs.
```
2019-01-09 13:58:27.476 [ERR] GRPC: Unary method /walletrpc.WalletLoaderService/OpenWallet invoked by 127.0.0.1:42788 errored: rpc error: code = AlreadyExists desc = loader.OpenExistingWallet: item already exists: wallet already opened
2019-01-09 13:58:30.070 [INF] GRPC: Unary method /walletrpc.WalletService/BestBlock invoked by 127.0.0.1:42788
2019-01-09 13:58:30.086 [INF] GRPC: Streaming method /walletrpc.WalletLoaderService/SpvSync invoked by 127.0.0.1:42788
2019-01-09 13:58:30.111 [INF] SYNC: Headers synced through block 0000000005d7a1c20d8ff4558ca678fb14fb458d9551017e9ac4a98a66415ffc height 107450
2019-01-09 13:58:30.112 [INF] SYNC: Transactions synced through block 0000000005d7a1c20d8ff4558ca678fb14fb458d9551017e9ac4a98a66415ffc height 107450
2019-01-09 13:58:31.581 [INF] SYNC: New peer 188.166.43.247:19108 /dcrwire:0.3.0/dcrd:1.5.0(pre)/ SFNodeNetwork|SFNodeCF
2019-01-09 13:58:31.607 [INF] CMGR: 25 addresses found from DNS seed x5.testnet-seed.decred.mindcry.org
2019-01-09 13:58:31.658 [INF] SYNC: New peer 45.55.94.2:19108 /dcrwire:0.3.0/dcrd:1.5.0(pre)/ SFNodeNetwork|SFNodeCF
2019-01-09 13:58:31.726 [INF] CMGR: 23 addresses found from DNS seed x5.testnet-seed.decred.org
2019-01-09 13:58:33.403 [INF] WLLT: Loading active addresses and unspent outputs...
2019-01-09 13:58:33.469 [INF] SYNC: New peer 100.24.245.163:19108 /dcrwire:0.3.0/dcrd:1.3.0/ SFNodeNetwork|SFNodeCF
2019-01-09 13:58:33.469 [INF] SYNC: New peer 100.24.142.141:19108 /dcrwire:0.3.0/dcrd:1.3.0/ SFNodeNetwork|SFNodeCF
2019-01-09 13:58:33.472 [INF] SYNC: New peer 13.78.146.191:19108 /dcrwire:0.3.0/dcrd:1.3.0/ SFNodeNetwork|SFNodeCF
2019-01-09 13:58:33.478 [INF] SYNC: New peer 13.78.146.191:19108 /dcrwire:0.3.0/dcrd:1.3.0/ SFNodeNetwork|SFNodeCF
2019-01-09 13:58:33.545 [INF] WLLT: Registered for transaction notifications for 141 address(es) and 12 output(s)
```

**Comment by ademuanthony** [comment]
I was not able to reproduce this issue with `godcr --mode=http` and `usewalletrpc=1` set in config

**Comment by itswisdomagain** [comment]
This appears to have been fixed in `dcrwallet version 1.5.0-pre+dev (Go version go1.11.4)`. That is currently pre-release and can be gotten by building `dcrwallet` from the source code on [dcrwallet master](https://github.com/decred/dcrwallet).
